test/commonModal.test.js

```diff
@@ -0,0 +1,40 @@
+import { getByTestId } from '@testing-library/dom';
+import '@testing-library/jest-dom';
+import {
+  hideElement,
+  renderComponentIntoSelectedTagID,
+} from '../src/utils/functions';
+import { CommonModal } from '../src/components/commonModal/CommonModal';
+import userEvent from '@testing-library/user-event';
+
+describe('Test common modal', () => {
+  beforeAll(() => {
+    document.body.innerHTML = `<div id="swquiz-app">
+      <div id="app-structure">
+        <div id="swquiz-header" class="header">
+        </div>
+      </div>
+    </div>`;
+    renderComponentIntoSelectedTagID(
+      CommonModal('kotek', hideElement),
+      'swquiz-app',
+    );
+  });
+
+  it('Should to render correctly', () => {
```

Comment:
```suggestion
  it('should be visible', () => {
```
albo 

 it('should render modal component', () => {